data-share/data-share-service/src/main/java/io/mosip/datashare/controller/DataShareController.java

```diff
@@ -70,10 +72,11 @@ public class DataShareController {
 			@ApiResponse(responseCode = "403", description = "Forbidden" ,content = @Content(schema = @Schema(hidden = true))),
 			@ApiResponse(responseCode = "404", description = "Not Found" ,content = @Content(schema = @Schema(hidden = true)))})
 	public ResponseEntity<Object> createDataShare(@RequestBody MultipartFile file,
-			@PathVariable("policyId") String policyId, @PathVariable("subscriberId") String subscriberId) {
-		
+			@PathVariable("policyId") String policyId, @PathVariable("subscriberId") String subscriberId,
+			@Parameter(description = "Number of allowed transaction in standalone mode") @RequestParam(required = false, name = "transactionsAllowed") String transactionsAllowed) {

```

Comment:
should give some other name for this query parameter like usageCount, accessAllowCount or accessAllowedFor or accessAllowedTimes or allowAccessCount ?

Reply:
since it works only with standalone mode, you can also try to keep that in the name, so it will avoid confusions when some is trying to use this query param in integrated mode

Reply:
Provide the query param as **"usageCountForStandaloneMode"**